ncm-metaconfig/src/main/metaconfig/udev/pan/rule.pan

```diff
@@ -0,0 +1,62 @@
+unique template metaconfig/udev/rule;
+
+@documentation{ 
+This example shows how to create a multiple rules. Please refer to schema for data structure.
+
+SELF[digit] : represents each rule in the file
+each rule consists of match_rule and assign_rule attributes
+   - each attribute can be dict of dict or dict of list of dict
+       - dict of list of dict allows mulitple matching or multiple assign with 
+         different keys (example of this below, 'attrs[0]' attribute or 'run[0]'
+         assign attribute)
+   - match_rule attributes could have operator of 2 types, ==, and != 
+   - assign_rule attributes could have operator of 3 types, =, := and += 
+
+Refer to manpage udev(7) for keys.
+            
+Rules to generate:
+#1 rule: ACTION=="add|change", ATTRS{device}=="0x1572", ATTRS{vendor}=="0x8086", DRIVERS=="i40e", \
```

Comment:
Ah slight snag which I didn't think of, `}` ends the annotation, so everything after this is a syntax error.

Reply:
`&lbrace;vendor&rbrace;` would be accepted and make it into the documentation at the expense of code readability

Reply:
updated as suggested

Reply:
not familiar with the tests being failed, could you help ?

Reply:
@jrha  any chance you can help with the testing?